optimization.md

```
@@ -0,0 +1,56 @@
+    absorption_min: 2.5
+    absorption_max: 3.5
+    tx_ref_rssi_min: -70
+    tx_ref_rssi_max: -50
+    rx_adj_rssi_min: -15
+    rx_adj_rssi_max: 20
+```
+
+The limits section will clamp the adjustment values the optimizer can set.
+
+
+## Help write this documentation!  Click the edit this page below.
```

Markdownlint

Punctuation: '.'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)